src/Monolog/Logger.php

```diff
@@ -312,7 +312,7 @@ public function addRecord(int $level, string $message, array $context = []): boo
 
         try {
             foreach ($this->processors as $processor) {
-                $record = call_user_func($processor, $record);
+                $record = $processor($record);
```

Comment:
I believe this is the place where we might see the results of this.. all the others are not offering much in performance.. just in order to be consistent.